src/main/config/feature.h

```diff
@@ -53,7 +53,7 @@ typedef enum {
     //FEATURE_SOFTSPI = 1 << 26, (removed)
     FEATURE_ESC_SENSOR = 1 << 27,
     FEATURE_ANTI_GRAVITY = 1 << 28,
-    FEATURE_DYNAMIC_FILTER = 1 << 29,
+    //FEATURE_DYNAMIC_FILTER = 1 << 29, (removed)
```

Comment:
Would there be benefit in tracking the version of this enum, to enable potential reuse? Seems a bit clunky to have to preserve old bit definitions.

Reply:
The features are used as a bitmask. Would this not add some complications if we wanted to add version control? It's already a pain in the configurator 😄 

But I don't have the experience to answer this.